class.ext_update.php

```diff
@@ -38,61 +40,34 @@ class ext_update
      *
      * @return bool Should the update option be shown?
      */
-    public function access()
+    public function access(): bool
     {
         if (count($this->getMetadataConfig())) {
             return true;
-        } elseif ($this->oldIndexRelatedTableNames()) {
+        }
+        if ($this->oldIndexRelatedTableNames()) {
             return true;
-        } elseif ($this->solariumSolrUpdateRequired()) {
+        }
+        if ($this->solariumSolrUpdateRequired()) {
             return true;
-        } elseif (count($this->oldFormatClasses())) {
+        }
+        if (count($this->oldFormatClasses())) {
```

Comment:
This test will always be run --> see below in https://github.com/kitodo/kitodo-presentation/pull/499/files#r404744493

Reply:
also fixed